File: foundations/py/6/chartype.py
ch = input('Enter a character:')

if ch.isupper():
    print(ch, 'is an upper case letter.')
elif ch.islower():
    print(ch, 'is a lower case letter.')
elif ch.isdigit():
    print(ch, 'is a digit.')
else:
    print(ch, 'is a non-alphanumeric character.')

File: foundations/cpp/10/chartype.cpp
#include <iostream>
using namespace std;

int main()
{
	char ch;
	cout << "Enter a character: ";
	cin >> ch;

	if (islower(ch))
		cout << ch << " is a lower case letter." << endl;
	else if (isupper(ch))
		cout << ch << " is an upper case letter." << endl;
	else if (isdigit(ch))
		cout << ch << " is a digit." << endl;
	else
		cout << ch << " is a non-alphanumeric character." << endl;
	return 0;
}